mdal/frmts/mdal_xmdf.hpp

```diff
@@ -29,7 +29,7 @@ namespace MDAL
    * are stored in single
    *   3D arrays (time, x, y) for vector datasets
    *   2D arrays (time, x) for scalar datasets
```

Comment:
please fix https://github.com/lutraconsulting/MDAL/blob/master/README.md the description of XMDF*: format, add that it also supports hydro-as